perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -41,6 +41,11 @@ sub disambiguate_location {
 
 sub admin_user_domain { 'northumberland.gov.uk' }
 
+sub is_defect {
+    my ($self, $p) = @_;
+    return $p->user_id == $self->body->comment_user_id;
```

Comment:
This is fine, and matches Northamptonshire, I'm just wondering if there's a reason we don't use `$p->service eq 'Open311'` (which is set for all pulled in reports, as far as I can see).

Reply:
Nice, I hadn't spotted that. Change made.